devops/scripts/deploy-public-upstream.sh

```diff
@@ -47,24 +50,40 @@ for commit in "${commits[@]}"; do
   if [[ -z "$commit" ]] ; then
     continue
   fi
+  # Sync commits from release (pantheon-systems/drupal-recommended) to release-source (pantheon-recommended/drupal-recommended).
```

Comment:
Is `pantheon-recommended/drupal-recommended` supposed to be `pantheon-upstream/drupal-recommended`, or is the purpose here to make another branch in `pantheon-systems/drupal-recommended` to track just the release commits?